src/Command/ContentHubUpgradeStart.php

```diff
@@ -95,10 +114,52 @@ public function __construct(EventDispatcherInterface $dispatcher, PlatformComman
     $this->platformCommandExecutioner = $platform_command_executioner;
   }
 
+  /**
+   * Sets the amplitude client property.
+   *
+   * @param \Acquia\Console\ContentHub\Client\AmplitudeClient $amplitude_client
+   *   Amplitude Client.
+   */
+  public function setAmplitudeClient(AmplitudeClient $amplitude_client) {
+    $this->amplitudeClient = $amplitude_client;
+  }
+
+  /**
+   * {@inheritDoc}
+   */
+  public function initialize(InputInterface $input, OutputInterface $output) {
```

Comment:
I am not familiar with the ticket and goal here but I took a quick look and this function has a lot of logic within multiple nested if and else statements. Would it be possible to simplify a bit or outsource into another function?

Reply:
Outsourced into another function.